### DISCUSSION TITLE
How to set Authorization: ApiKey {ApiKey} in Elsa Server Side?

### DISCUSSION THREAD

**Discussion by sagegz** [elsa-workflows/elsa-core#5190] [Q&A]
Where's the node in appsettings.json to set ApiKey in Elsa Server Side?

**Comment by sfmskywalker** [comment]
That would be here: https://github.com/elsa-workflows/elsa-core/blob/main/src/bundles/Elsa.Server.Web/appsettings.json#L76

**Reply by sagegz** [reply]
Thanks for your reply, but which algorithm do you hash the key?I couldn't find the original key.Can I set the original key in json file directly? 
Based on my understanding, it's a bit more difficult to finish the official documents because it lacks coherent details.@sfmskywalker
